shell.nix

```diff
@@ -0,0 +1,14 @@
+with import <nixpkgs> {};
```

Comment:
Thanks for this.

I'm a Flake fan myself (and will probably add that next so we can pin dependencies) but any Nix is better than none. :)